include/reactor-uc/environment.h

```diff
@@ -9,6 +9,7 @@
 #include "reactor-uc/scheduler.h"
 
 typedef struct Environment Environment;
+extern Environment *_lf_environment;
```

Comment:
I created this global backdoor so that the logging functions could read out the current time and add it to the printouts... we could possibly use this also within the TcpIpChannel and UdpCoapChannel

Reply:
So you mean we remove the `Environment pointer` from the constructor and just use the global pointer?

Reply:
Yes that is possible. It is a little dirty, but, having global access to the environment also makes sense.